### PR TITLE
[1LP][RFR][NOTEST] Corrected table xpath and added quotas property

### PR DESCRIPTION
**Pull request by vatsalparekh**
Purpose or Intent
=================

Currently there are no tests checking this table or the property so I've put [NOTEST]

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 4f92d13f454b99e20777e9538303367669a85258:

`cfme/configure/access_control.py`:
- :bangbang: Line 1414:18: **E203** *whitespace before ':'*
- :bangbang: Line 1425:29: **E126** *continuation line over-indented for hanging indent*
- :bangbang: Line 1429:25: **E121** *continuation line under-indented for hanging indent*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

File: cfme/configure/access_control.py [hunk 1]
```diff
@@ -1405,6 +1407,29 @@ def set_quota(self, **kwargs):
...
+
+        for field in quotas.keys():
+            for item in view.table.rows():
+                if item.name.text == quotas[field]:
+                    quotas[field] = {
+                        'total': item.total_quota.text,
+                        'in_use': item.in_use.text,
+                        'allocated': item.allocated.text,
+                        'available': item.available.text
+                    }
```
Comment (mfalesni) [inline cfme/configure/access_control.py]: I think you can replace the for + if with this:
```python
item = view.table.row(name=quotas[field])
```
It should be faster and more obvious.

Comment (vatsalparekh) [reply cfme/configure/access_control.py]: Changed.

Comment (mfalesni) [reply cfme/configure/access_control.py]: Why the `for` is still there? The code hints that it should only process one row, why?

Comment (vatsalparekh) [reply cfme/configure/access_control.py]: @mfalesni Corrected.

File: cfme/configure/access_control.py [hunk 2]
```diff
@@ -1289,7 +1289,9 @@ class DetailsTenantView(ConfigurationView):
     name = Text('Name')
     description = Text('Description')
     parent = Text('Parent')
-    table = Table('//*[@id="fieldset"]/table')
+    table = Table(VersionPick(
+        {Version.lowest(): '//*[@id="fieldset"]/table',
```
Comment (izapolsk) [inline cfme/configure/access_control.py]: REQUIRED: I believe you can get rid of VersionPick here by using something like
```
//*[self::fieldset or @id="fieldset"]/table
```

File: cfme/configure/access_control.py [hunk 3]
```diff
@@ -1405,6 +1407,28 @@ def set_quota(self, **kwargs):
...
+        view = navigate_to(self, 'Details')
+        quotas = {
+            'cpu': 'Allocated Virtual CPUs',
+            'memory': 'Allocated Memory in GB',
+            'storage': 'Allocated Storage in GB',
+            'num_vms': 'Allocated Number of Virtual Machines',
+            'templates': 'Allocated Number of Templates'
+        }
+
+        for field in quotas.keys():
```
Comment (izapolsk) [inline cfme/configure/access_control.py]: OPTIONAL: when you iterate over dict, python uses keys() be default.
So,
for key in some_dict.keys()  equal to for key in some_dict

File: cfme/configure/access_control.py (hunk 2 above)
Comment (mfalesni) [reply cfme/configure/access_control.py]: Depends how many things match that xpath, in the case of fieldset tag I would not be so sure about uniqueness, I prefer an ID in the path :)

Comment (izapolsk) [reply cfme/configure/access_control.py]: @mfalesni,  as far as I see old div with fieldset id turned into fieldset class. So, I doubt there are several fieldset classes on the page.
One more note, such long xpath with indexes and etc doesn't look robust.

Comment (vatsalparekh) [reply cfme/configure/access_control.py]: I checked on all z-streams, this table is the only one having `fieldset` on the page

**Pull request merged**